lstchain/scripts/lstchain_dl1_to_dl2.py

```diff
@@ -46,65 +47,54 @@
 parser = argparse.ArgumentParser(description="DL1 to DL2")
 
 # Required arguments
-parser.add_argument('--input-file', '-f', type=str,
-                    dest='input_file',
-                    help='path to a DL1 HDF5 file',
-                    default=None, required=True)
-
-parser.add_argument('--path-models', '-p', action='store', type=str,
+parser.add_argument('--input-files', '-f',
+                    type=Path,
+                    nargs='+',
```

Comment:
what is this `nargs` changing in the parser?

Reply:
You get a list of files but at least one

Reply:
See https://docs.python.org/3/library/argparse.html#nargs

Reply:
thanks!